api-common/src/main/kotlin/io/qalipsis/api/meters/MeterType.kt

```diff
@@ -17,18 +17,20 @@
 package io.qalipsis.api.meters
 
 /**
- * Possibilities of [io.qalipsis.api.meters] that are available.
+ * Possibilities of [io.qalipsis.api.meters.Meter] that are available.
```

Comment:
```suggestion
 * Possibilities of [io.qalipsis.api.meters.Meter] that is available.
```

Reply:
But Possibilities is plural, right?

Reply:
Yea. I think I was looking at "Meter" as singular when I wrote that. But thinking about it again it's fine.